docs/examples/basic-ssh-username-private-key-passphrase.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Secret
+metadata:
+# this is the jenkins id.
+  name: "jenkins-key"
+  labels:
+# so we know what type it is.
+    "jenkins.io/credentials-type": "basicSSHUserPrivateKey"
```

Comment:
Suggest using `sshUserPrivateKey` to match the binding symbol in `withCredentials`.

Reply:
I'm ok to rename it, but should we also rename every file/class/test for consistency ?

Reply:
Note that the _existing_ types in this plugin do not match the symbols established in `credentials-binding`. I suppose if we wished to establish consistency we could continue to accept the current `jenkins.io/credentials-type`s but deprecate them and only document the new ones.

Reply:
@jglick an `@Symbol` referes to a conrete `binding` class.  That class can (and does) bind base classes that could be extended.  Hypothetically we could have an SSH key that has a URL instead of encoded key and the credentials-binding would still be able to bind it (and the credential implementation knows how to retreive that key from the URL), however we (this plugin) would not be able to create it as we need to bind to the specific implementation.  
Basically the Credential *implementation* class and the Credential *Binding* are 2 separate disjoint things.